UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.LoadTests/TestCase/CachePublishPTWithLiveData.js

```diff
@@ -0,0 +1,130 @@
+import http from 'k6/http';
+import { check,sleep,metrics,output } from 'k6';
+import { vu,scenario,instance } from 'k6/execution';
+import exec from 'k6/execution';
+import { htmlReport } from "https://raw.githubusercontent.com/benc-uk/k6-reporter/main/dist/bundle.js";
+import { textSummary } from "https://jslib.k6.io/k6-summary/0.0.1/index.js";
+import { Trend } from 'k6/metrics';
+
+const publishEvent = new Trend('NewFilePublishEvent');
+const TestData = require('../scripts/CacheTestData.js')
+const config=JSON.parse(open('./config.json'));
+const publishLogFile = JSON.parse(open('../LiveLogs/FilePublishedLogs.json'));
+const ESSPublishFile = JSON.parse(open('../LiveLogs/ESSPublishedLogs.json'));
+const logParser=require('../helper/LogParser.js');
+const ess=require('../scripts/ReplayRequest.js');
+let retryCount=0;
+
+export let options = {
+  scenarios: {
+
+    'NewPublish_400PerHour':{
+      executor: 'constant-arrival-rate',
+      duration:'1m',
```

Comment:
Doesn't looks correct, Please check in other loads as well and correct it.

Reply:
Resolved